lib/pos_epoch.go

```diff
@@ -14,7 +14,10 @@ import (
 
 type EpochEntry struct {
 	EpochNumber      uint64
-	FinalBlockHeight uint64
+	StartBlockHeight uint64
+	StartViewNumber  uint64
```

Comment:
"XYZView" is an overloaded term I guess. It would be ideal if we can rename this to InitialView for consistency with the rest of the codebase (or alternatively even InitialFastHotStuffView if it leads to confusion)
```suggestion
	InitialView  uint64
```